File: src/back/BLL/Dtos/EditShopDto.cs
using back.BLL.Dtos.HelpModels;

namespace back.BLL.Dtos
{
    public class EditShopDto
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Address { get; set; }
        public List<int>? Categories { get; set; }
        public List<WorkingHoursDto>? WorkingHours { get; set; }
        public int? PIB { get; set; }
        public string? AccountNumber { get; set; }
    }
}

File: src/Front/app/src/main/java/com/example/front/model/DTO/EditShopDTO.kt
package com.example.front.model.DTO

import com.google.gson.annotations.SerializedName

data class EditShopDTO(
    @SerializedName("id") val id: Int,
    @SerializedName("name") val name: String?,
    @SerializedName("address") val address: String?,
    @SerializedName("categories") val categories: List<Int>?,
    @SerializedName("workingHours") val workingHours: List<WorkingHoursNewShopDTO>?,
    @SerializedName("pib") val pib: Int?,
    @SerializedName("accountNumber") val accountNumber: String?
)